src/main/kotlin/net/atos/zac/app/signalering/SignaleringRestService.kt

```diff
@@ -100,41 +101,21 @@ class SignaleringRestService @Inject constructor(
                 .build()
         }
 
+    private fun Long.maxPages(pageSize: Int) = (this + pageSize - 1) / pageSize
```

Comment:
Is this still used?

Reply:
It's used for zaken paging only. It still makes sense imho to have it in a separate function to self-comment what this calculations do.